### ISSUE TITLE
Notification for update on submitted templates

### ISSUE THREAD

**Issue by mdshamoon** [glific/glific#1777]
@mdshamoon we could do it this way 
![image](https://user-images.githubusercontent.com/16714604/133998695-cd8c211e-67de-4147-8080-4db35e67a12a.png)
just to draw attention to the templates section.

_Originally posted by @abhi1203 in https://github.com/glific/glific-frontend/issues/1658#issuecomment-922864473_

Need backend support to get an API that tells whether a template has been approved or rejected recently.

**Comment by pankaj-ag** [comment]
We will create a new notification if the HSM template status changes. Let me know if that works.

**Comment by kurund** [comment]
@pankaj-ag 

Any update on this?

**Comment by pankaj-ag** [comment]
@kurund @AkhileshNegi look at work on this today.

**Comment by pankaj-ag** [comment]
@kurund I think We need to add one more category in notifications with the name Info since this notification does not fall under the critical/warning category.

Let me know if you have any thoughts on that.

**Comment by dlobo** [comment]
yes, lets create an information category and add this notification there., now that we have a notification framework. Lets knock this issue out

**Comment by pankaj-ag** [comment]
@glific/glific-frontend-team we have added one more category called information. We can now show it on the frontend side.
